scientist-softserv/webstore
# 362-requests-specs

# Story
stubbed requests cypress e2e tests are passing **locally**. they will still fail in ci due to configuration. (ref: https://github.com/scientist-softserv/webstore/issues/378)

- ref: #362
# Screenshots / Video
<img width="958" alt="image" src="https://github.com/scientist-softserv/webstore/assets/29032869/4c2f4887-252a-46a0-a518-8afc0448f99c">


# Testing
1. pull this branch
2. ensure you have all of the environment variables in the `env` property of "cypress.config.js" set up.
3. run `yarn cypress:e2e`
4. choose the "requests" test
